src/Enzo/enzo-core/EnzoProblem.cpp

```diff
@@ -686,13 +686,8 @@ Method * EnzoProblem::create_method_
 
   } else if (name == "mhd_vlct") {
 
-    method = new EnzoMethodMHDVlct
-      (enzo_config->method_vlct_riemann_solver,
-       enzo_config->method_vlct_time_scheme,
-       enzo_config->method_vlct_reconstruct_method,
-       enzo_config->method_vlct_theta_limiter,
-       enzo_config->method_vlct_mhd_choice,
-       store_fluxes_for_corrections);
+    method = new EnzoMethodMHDVlct(p_group, store_fluxes_for_corrections);
+    skip_auto_courant = true;
```

Comment:
I don't see any other references to this variable in this PR. What is its purpose?

Reply:
Enzo-E has traditionally read and set the courant factor of every method automatically. However, we are moving away from that. Now, Methods read in their own courant factor (since different methods have different requirements - e.g. it should never exceed 0.5 for vl+ct). This variable is being used to indicate that courant factor should not automatically be set